packages/template-ui/src/components/Breadcrumb.vue

```diff
@@ -1,28 +1,43 @@
 <template>
   <b-navbar-nav
-    v-if="(node.ancestors && node.ancestors.length) || notAtHome"
     class="mt-3"
     aria-label="breadcrumb"
   >
-    <ChannelLogo class="mt-1" :channel="channel" size="sm" />
+    <ChannelLogo v-if="!xs" class="mt-1" :channel="channel" size="sm" />
```

Comment:
This is something that can be also done with the bootstrap [display classes](https://getbootstrap.com/docs/4.6/utilities/display/#hiding-elements).

The use of `responsiveMixin` looks simpler, but I'm not sure if we should try to use the bootstrap standard classes when it's possible.

Reply:
Oh yes! Indeed we should use the display class whenever possible. Thanks!